casanovo/casanovo.py

```diff
@@ -378,7 +378,7 @@ def setup_model(
     for key, value in config.items():
         logger.debug("%s = %s", str(key), str(value))
 
-    return config
+    return config, model
```

Comment:
**question:** Why does the model need to be returned?

Reply:
Because the automatically downloaded weights aren't used otherwise - they're returned with `_get_model_weights()` inside the function.